### PR TITLE
Keep more of GitHub release notes body

### PR DESCRIPTION
**Pull request by john-kurkowski**
Only omit the first paragraph, "What's Changed." Keep the e.g. "New Contributors" paragraph and "Full Changelog" link.

### CODE REVIEW

**Comment by john-kurkowski** [comment]
FYI @ekcorso

File: scripts/release.py [hunk 1]
```diff
@@ -131,25 +124,15 @@ def get_changelog_release_notes(release_notes_url: str, version: str) -> str:
     if match:
         return str(match.group(1)).strip()
     else:
-        print(
-            f"WARNING: Failed to parse changelog release notes. Manually copy this version's notes from the CHANGELOG.md file to {release_notes_url}.",
```
Comment (john-kurkowski) [inline scripts/release.py:135]: Haven't tripped on this error handler yet, but I think the URL in it was wrong. It's a _compare_ link like https://github.com/john-kurkowski/tldextract/compare/5.1.1...5.1.2, not a create-release link. I moved this error print to after we have the release link, with a hardcoded URL.

File: scripts/release.py [hunk 2]
```diff
@@ -100,24 +104,13 @@ def generate_github_release_notes_body(token: str, version: str) -> str:
...
-        return match.group(1)
-    else:
-        print(
-            "WARNING: Failed to parse release notes URL from GitHub response.",
-            file=sys.stderr,
-        )
-        return ""
+    body = str(response.json()["body"])
+    paragraphs = body.split("\n\n")
+    return "\n\n".join(paragraphs[1:])
```
Comment (ekcorso) [inline scripts/release.py:110]: So concise and it got rid of the REGEX-- love it. Still a little brittle if this format changes, but I think this is a great improvement.

File: scripts/release.py (hunk 1 above)
Comment (ekcorso) [reply scripts/release.py:135]: Yeah, good point. I think I garbled this warning when I added the `release_notes_url`. Your new placement is correct: it should be directing the user to the url of the newly created draft release to manually add the notes. Thanks for catching this!

**Review by ekcorso — APPROVED** [review]

**Pull request merged**